### ISSUE TITLE
Broken w/r/t to keyboard navigation

### ISSUE THREAD

**Issue by bhaibel** [easepick/easepick#70]
**Describe the bug**
When I try to use the demo datepicker in the documentation site with a keyboard, I can tab to the "previous month" and "next month" links but I cannot perform any other tasks. In particular, I cannot actually select dates.

**To Reproduce**
1. Go to https://easepick.com/
2. Press the tab key several times
4. Note that you can't 'reach' the main datepicker area.

**Expected behavior**
I expect to be able to tab into the datepicker as a whole and select dates with the keyboard.

**Code Playground**
n/a

**Additional context**
This is a showstopper issue for accessibility compliance.

**Comment by wakirin** [comment]
In fact the picker on index page doesn't included KbdPlugin, that is why keyboard navigation doesn't work.
You can check an example on this page: https://easepick.com/packages/kbd-plugin

(I will add KbdPlugin to the picker on the main page later)

**Comment by wakirin** [comment]
fixed
